app/Http/Controllers/SearchResultController.php

```diff
@@ -9,7 +9,7 @@ class SearchResultController extends Controller
 {
     public function displayResults() {
         return view('search-results', [
-            'title'=>$this->gameService->searchGamesByTitle()
+            'title'=>$this->GameService->searchGamesByTitle()
```

Comment:
i'm not sure why we have this SearchResultController, seems like this could be on the GameController. regardless, this won't work until you add GameService as a dependency in the constructor, like what is already being done in the GameController.

Reply:
also you will need to pass a title to this method to search by. since the route is a GET you should be looking in the request query params

Reply:
Haha yeah that makes perfect sense in retrospect about not needing SearchResultController. I deleted that and moved the method over to GameController and the search-results page is now displaying. Just need to work on displaying the results and I think this issue should be pretty much done! Will be back again soon hopefully with a push that completes that.